app/controllers/concerns/unique_resource_url.rb

```diff
@@ -0,0 +1,9 @@
+module UniqueResourceUrl
+  extend ActiveSupport::Concern
+
+  def ensure_or_redirect_to(resource)
```

Comment:
would it be more descriptive to go with `ensure_path_or_redirect_to` ?

Reply:
Yes, indeed!